iosxr_iso2vbox.py

```diff
@@ -391,87 +416,39 @@ def main(argv):
                         action='store_const', const=logging.DEBUG,
                         default=logging.INFO, help='turn on verbose messages')
 
-    args = parser.parse_args()
-
-    # Handle Input ISO (Local or URI)
-    if re.search(':/', args.ISO_FILE):
-        # URI Image
-        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
-        logger.debug('Will attempt to scp the remote image to current working dir. You may be required to enter your password.')
-        logger.debug('%s\n', cmd_string)
-        subprocess.call(cmd_string, shell=True)
-        input_iso = os.path.basename(args.ISO_FILE)
-    else:
-        # Local image
-        input_iso = args.ISO_FILE
-
-    # Handle create OVA
-    create_ova = args.create_ova
+    return parser.parse_args()
 
-    if not os.path.exists(input_iso):
-        sys.exit('%s does not exist' % input_iso)
 
-    # Set Virtualbox VM name from the input ISO
-    vmname = os.path.basename(os.path.splitext(input_iso)[0])
-
-    set_logging()
-    logger.setLevel(level=args.verbose)
-
-    logger.debug('Input ISO is %s', input_iso)
+def create_vbox_vm(vmname, base_dir, input_iso):
+    """Create and configure (but do not start) the VirtualBox VM."""
+    logger.info('Creating and configuring VirtualBox VM')
 
     # Set the RAM according to mini of full ISO
     if 'mini' in input_iso:
         ram = 3072
-        logger.debug('%s is a mini image, RAM allocated is %s MB', input_iso, ram)
+        logger.debug('%s is a mini image, RAM allocated is %s MB',
+                     input_iso, ram)
```

Comment:
Do you need to wrap these lines? I think for the most part I ignored the 80-column rule, so not sure if there'll be a quite a few more to wrap if you start doing it.